Cargo.lock

```diff
@@ -1,1209 +0,0 @@
-dependencies = [
- "thiserror",
- "tracing",
- "wiggle-macro",
- "witx",
-]
-
-[[package]]
-name = "wiggle-generate"
-version = "0.19.0"
-source = "git+https://github.com/bytecodealliance/wasmtime.git?rev=8ac4bd1d0d8228b97a88b1841cfc0247e9ef4306#8ac4bd1d0d8228b97a88b1841cfc0247e9ef4306"
-dependencies = [
- "anyhow",
- "heck",
- "proc-macro2",
- "quote",
- "syn",
- "witx",
-]
-
-[[package]]
-name = "wiggle-macro"
-version = "0.19.0"
-source = "git+https://github.com/bytecodealliance/wasmtime.git?rev=8ac4bd1d0d8228b97a88b1841cfc0247e9ef4306#8ac4bd1d0d8228b97a88b1841cfc0247e9ef4306"
-dependencies = [
- "quote",
- "syn",
- "wiggle-generate",
- "witx",
-]
-
-[[package]]
-name = "winapi"
-version = "0.3.9"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "5c839a674fcd7a98952e593242ea400abe93992746761e38641405d28b00f419"
-dependencies = [
- "winapi-i686-pc-windows-gnu",
- "winapi-x86_64-pc-windows-gnu",
-]
-
-[[package]]
-name = "winapi-i686-pc-windows-gnu"
-version = "0.4.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6"
-
-[[package]]
-name = "winapi-util"
-version = "0.1.5"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "70ec6ce85bb158151cae5e5c87f95a8e97d2c0c4b001223f33a334e3ce5de178"
-dependencies = [
- "winapi",
-]
-
-[[package]]
-name = "winapi-x86_64-pc-windows-gnu"
-version = "0.4.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f"
-
-[[package]]
-name = "winx"
-version = "0.19.0"
-source = "git+https://github.com/bytecodealliance/wasmtime.git?rev=8ac4bd1d0d8228b97a88b1841cfc0247e9ef4306#8ac4bd1d0d8228b97a88b1841cfc0247e9ef4306"
-dependencies = [
- "bitflags",
- "cvt",
- "winapi",
-]
-
-[[package]]
-name = "witx"
-version = "0.8.7"
-source = "git+https://github.com/bytecodealliance/wasmtime.git?rev=8ac4bd1d0d8228b97a88b1841cfc0247e9ef4306#8ac4bd1d0d8228b97a88b1841cfc0247e9ef4306"
-dependencies = [
- "anyhow",
- "log",
- "thiserror",
- "wast 22.0.0",
-]
-
-[[package]]
-name = "xattr"
-version = "0.2.2"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "244c3741f4240ef46274860397c7c74e50eb23624996930e484c16679633a54c"
-dependencies = [
- "libc",
-]
-
-[[package]]
-name = "yaml-rust"
-version = "0.4.4"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "39f0c922f1a334134dc2f7a8b67dc5d25f0735263feec974345ff706bcf20b0d"
-dependencies = [
- "linked-hash-map",
-]
-
-[[package]]
-name = "yanix"
-version = "0.19.0"
-source = "git+https://github.com/bytecodealliance/wasmtime.git?rev=8ac4bd1d0d8228b97a88b1841cfc0247e9ef4306#8ac4bd1d0d8228b97a88b1841cfc0247e9ef4306"
-dependencies = [
- "bitflags",
- "cfg-if",
- "filetime",
- "libc",
- "tracing",
-]
```

Comment:
Don't delete `Cargo.lock`. We check in the lockfile for binaries but not for libraries.

Reply:
OK.  My bad - could you explain why?